### PR TITLE
Display lock time next to UTXO amount

### PR DESCRIPTION
**Pull request by nop33**
https://user-images.githubusercontent.com/1579899/161769226-8b8c8a45-9449-48f1-b9da-1321956f460e.mp4

### CODE REVIEW

File: src/components/Table/TableBody.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export default styled(TableBody)`
         : ''}
 
     &.details div {
-      overflow: hidden;
+      /* overflow: hidden; */
```
Comment (nop33) [inline src/components/Table/TableBody.tsx:63]: @mvaivre I am not sure if this breaks something, that's why I commented on it to discuss it. I haven't seen any problems, I wonder if you remember why this is here.

File: src/components/LockTime.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+import dayjs from 'dayjs'
+import { Lock } from 'lucide-react'
+
+interface LockTimeProps {
+  timestamp: number
+  className?: string
+}
+
+const LockTime = ({ timestamp, className }: LockTimeProps) => {
```
Comment (mvaivre) [inline src/components/LockTime.tsx:27]: What about calling it `LockTimeIcon` for explicitness?

File: src/components/Table/TableBody.tsx (hunk 1 above)
Comment (mvaivre) [reply src/components/Table/TableBody.tsx:63]: Yep, if you don't hide the overflow, there's a glitch when closing the details table.

https://user-images.githubusercontent.com/6715943/161777886-8def459c-b498-4484-acd3-0a9f9f0c87ab.mov

**Review by mvaivre — APPROVED** [review]
I took the liberty to rebase and add a tiny purely cosmetic commit.

**Pull request merged**